style/checkstyle.xml

```diff
@@ -242,5 +242,8 @@
             <property name="exceptionVariableName" value="expected"/>
         </module>
         <module name="CommentsIndentation"/>
+        <module name="IllegalImport">
```

Comment:
how do we deal with a third party method, that return these? force users to convert to `Option` using the helpers?

Reply:
Yes, right. We already provided `Option` and some utility converter methods, the code style should be consistency. Users can use those methods in the scenarios where return `java.util.Optional` or Guava Optional, e.g. calling java streaming API.